systemd/crc-test-vsock.py

```
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Ruff (0.14.0)

Shebang is present but file is not executable

(EXE001)